From 833afe3264b722551a3f5e9ad2e559cace7adb3c Mon Sep 17 00:00:00 2001
From: Evan Huus <eapache@gmail.com>
Date: Mon, 8 Sep 2014 08:11:40 -0400
Subject: [PATCH 5/6] ses: initialize pres_ctx_id in session struct

Bug: 10454
Change-Id: Ib381646cad0a039514117735c48b461c9950f705
Reviewed-on: https://code.wireshark.org/review/4033
Reviewed-by: Evan Huus <eapache@gmail.com>
(cherry picked from commit 433a444d148f86f2562f804d25a57d00dc277cc0)
Reviewed-on: https://code.wireshark.org/review/4036
(cherry picked from commit 0942b2c13362b2e0a07a4583b46b59db6b44d7a8)
Reviewed-on: https://code.wireshark.org/review/4037
Reviewed-on: https://code.wireshark.org/review/4426
Reviewed-by: Balint Reczey <balint@balintreczey.hu>
Tested-by: Balint Reczey <balint@balintreczey.hu>
---
 epan/dissectors/packet-ses.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/epan/dissectors/packet-ses.c b/epan/dissectors/packet-ses.c
index c442e68..18dd490 100644
--- a/epan/dissectors/packet-ses.c
+++ b/epan/dissectors/packet-ses.c
@@ -1044,6 +1044,7 @@ dissect_spdu(tvbuff_t *tvb, int offset, packet_info *pinfo, proto_tree *tree,
 	type = tvb_get_guint8(tvb, offset);
 	session.spdu_type = type;
 	session.abort_type = SESSION_NO_ABORT;
+	session.pres_ctx_id = 0;
 	session.ros_op = 0;
 	session.rtse_reassemble = FALSE;
 
-- 
2.1.1

